tile.yml

```diff
@@ -63,17 +63,18 @@ packages:
     env:
       GOPACKAGENAME: github.com/wavefronthq/cloud-foundry-nozzle-go
 
-      NOZZLE_API_URL: (( .properties.pcf_host.value ))
-      NOZZLE_USERNAME: (( .properties.pcf_user.value ))
-      NOZZLE_PASSWORD: (( .properties.pcf_password.value ))
-      NOZZLE_LOG_STREAM_URL: (( .properties.pcf_log_stream.value ))
+      NOZZLE_API_URL: https://api.(( ..cf.cloud_controller.system_domain.value ))
+      NOZZLE_USERNAME: admin
```

Comment:
The admin account has more permissions than what you need as a nozzle.  You may want to investigate whether you can create a UAA client with a more restricted set of scopes.

Reply:
Second this, probably should not hard code the username.
Our nozzle docs does recommend creating a dedicated UAA user with just the required perms:
https://docs.pivotal.io/partners/wavefront-nozzle/installing.html

Reply:
+1 Better to have the user specify a user and have details in docs on setting up scope.